src/c_console.cpp

```diff
@@ -285,6 +286,12 @@ CUSTOM_CVAR (Int, msg4color, 0, CVAR_ARCHIVE)
 	setmsgcolor (4, self);
 }
 
+CUSTOM_CVAR (Int, msg5color, 21, CVAR_ARCHIVE)
+{
+	// [AK] Private chat message color.
```

Comment:
Please move this comment to where the delta to GZDoom starts, i.e. before `CUSTOM_CVAR (Int, msg5color ....`